packages/react-components/src/constants/keyCodes.ts

```diff
@@ -1,3 +1,12 @@
 export const KeyCodes = {
   esc: 'Escape',
+  enter: 'Enter',
+  backspace: 'Backspace',
+  delete: 'Delete',
+  del: 'Del', // for IE 9-11, FF 36 and earlier
```

Comment:
I guess we don't support IE and old versions of FF.

Reply:
ok, I'll remove it then 👌